doc/sphinx/installation.rst

```diff
@@ -178,15 +186,22 @@ Quick installation
 If you have installed the requirements (see section :ref:`Requirements
 <requirements>` ) in standard locations, to compile, it is usually enough to
 create a build directory and call ``cmake`` and ``make`` (optional steps
-which modify the build process are commented out):
+which modify the build process are commented out).
+To chose the correct Python version, run
+.. code-block:: bash
```

Comment:
```suggestion

.. code-block:: bash

```
I think there should always be a newline character both before and after a `code-block` directive.